src/others/contexts/file-downloader.tsx

```diff
@@ -60,31 +62,36 @@ export const FileDownloaderContextProvider: FunctionComponent = ({ children }) =
 
   const filteredAidRequests = aidRequests?.filter(
     (aidRequest) =>
-      (activeCategoryFilters.length === 0 || activeCategoryFilters.includes(aidRequest.category_id)) &&
+      (activeCategoryFilters.length === 0 || activeCategoryFilters.includes(aidRequest.category_id as string)) && // typecasting necessary because filter item is string | boolean
```

Comment:
i'm confused here, maybe "as string" came from the time when .category_id was typed ad ID,
but now it has "string" type explicitly.
why exactly is extra conversion from string to string needed?

Reply:
removed